wiki/blocks/first-block.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Adding your first Block
+index: 1
+---
+# Adding your first Block
+
+Creating a Block is pretty similar to creating an Item, [read the corresponding article](../items/first-item) for more elaborate explanations.
+
+## Creating and registering the Block
+
+First we create the Block and store it in a field:
+
+```java
+public final Block EXAMPLE_BLOCK = new Block(new QuiltBlockSettings())
+```
+
+Then we register it in the `onInitialize()` function:
+
+```java
+Registry.register(Registries.BLOCK, new Identifier(mod.metadata().id(), "example_block"), EXAMPLE_BLOCK);
+```
+
+## Adding a BlockItem
+
+Having done that, we can place the block using the `setblock` command, but there isn't an item to place it. So we additionally register a `BlockItem` for the block and add it to the `BUILDING_BLOCKS` item group:
```

Comment:
```suggestion
Having done that, we can place the block using the `setblock` command but, opening the creative menu, we won't be able to find an item version of it. To fix this, we register a `BlockItem` for the block and add it to an item group, in our case `BUILDING_BLOCKS`:
```
cleaner language + avoiding the possibility of someone thinking that every block has to go in that group